app/models/user.rb

```diff
@@ -4,9 +4,10 @@ class User < ActiveRecord::Base
             format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i }
   has_secure_password
   has_many :orders
+  has_many :businesses
   has_many :order_items, through: :orders
   has_many :items, through: :order_items
-  enum role: %w(default admin)
+  enum role: %w(default admin business_admin)
```

Comment:
I agree - also as far as today goes, our current admin is the business admin. So the privleges wouldn't need to change.